framework_agreement/view/product_view.xml

```diff
@@ -1,83 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
   <data>
-    <record id="agreement in product view" model="ir.ui.view">
+    <record id="agreement_in_product_view" model="ir.ui.view">
       <field name="name">agreement in product view</field>
       <field name="model">product.product</field>
       <field name="inherit_id" ref="product.product_normal_form_view"/>
       <field name="arch" type="xml">
         <field name="seller_ids" position="after">
           <group string="Framework agreements (LTA)" colspan="4">
-            <field name="framework_agreement_ids" nolabel="1">
```

Comment:
There were some problems with this embedded view. It did not have some of the fields, such as the portfolio field so that I could not add a new agreement from this view.